lib/consul/migrate/cli.rb

```diff
@@ -0,0 +1,71 @@
+require 'consul/migrate/version'
+require 'consul/migrate/client'
+require 'consul/migrate/defaults'
+require 'thor'
+require 'yaml'
+require 'fileutils'
+
+module Consul
+  module Migrate
+    class Cli < Thor
+
+      attr_accessor :client
+
+      def initialize(*args)
```

Comment:
@maclennann I thought that having `read_config` in here would initialize a class instance of `@client` every time consul-migrate's CLI is executed so that when I execute `consul-migrate export` it would actually have initialized `@client`